### ISSUE TITLE
Underscores in interface names with specific schema definition.

### ISSUE THREAD

**Issue by Nalhin** [acacode/swagger-typescript-api#65]
Spring automatically generates schema definitions with generic types (see the example for details). 
This causes the typescript interfaces to contain underscores in place of "«" symbols when such a schema is used.

Current result:
*Page_TemplateResponseDto_*

Expected result:
*PageTemplateResponseDto*

OA version - 2.0

Example schema:
```json
{
   "Page«TemplateResponseDto»":{
      "type":"object",
      "properties":{
         "content":{
            "type":"array",
            "items":{
               "$ref":"#/definitions/TemplateResponseDto"
            }
         },
         "empty":{
            "type":"boolean"
         },
         "first":{
            "type":"boolean"
         },
         "last":{
            "type":"boolean"
         },
         "number":{
            "type":"integer",
            "format":"int32"
         },
         "numberOfElements":{
            "type":"integer",
            "format":"int32"
         },
         "pageable":{
            "$ref":"#/definitions/Pageable"
         },
         "size":{
            "type":"integer",
            "format":"int32"
         },
         "sort":{
            "$ref":"#/definitions/Sort"
         },
         "totalElements":{
            "type":"integer",
            "format":"int64"
         },
         "totalPages":{
            "type":"integer",
            "format":"int32"
         }
      },
      "title":"Page«TemplateResponseDto»"
   }
}
```

**Comment by js2me** [comment]
@Nalhin thanks for this issue!  
I'll fix that at next release (next day, GMT+3)

**Comment by js2me** [comment]
It should be fixed in 1.11.0 release
